openapi/connector_mgmt-private-admin.yaml

```diff
@@ -221,6 +221,18 @@ paths:
             type: string
           in: path
           required: true
+        - name: channel_updates
+          description: include only deployments that have channel updates
+          schema:
+            type: boolean
+          in: query
+          required: false
+        - name: dangling_deployments
```

Comment:
Can these two options be combined? If not, perhaps user should not be able to specify them both together in the query??

Reply:
well in theory you could ask for the dangling deployments that have a pending channel update. Most of the time it returns no items, but is a valid query I guess?

Reply:
ok, if they can be combined it's acceptable.